src/stories/HeatmapVis.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactElement } from 'react';
+import type { Story } from '@storybook/react/types-6-0';
+import ndarray from 'ndarray';
+import FillHeight from '../../.storybook/decorators/FillHeight';
+import HeatmapVis, {
+  HeatmapVisProps,
+} from '../h5web/visualizations/heatmap/HeatmapVis';
+import mockData from '../h5web/providers/mock/data.json';
+
+const dataset1 = mockData.datasets['c8f60c29-aae2-11ea-84a9-b94ddd2ec9e8'];
+
+const Template: Story<HeatmapVisProps> = (args): ReactElement => (
+  <HeatmapVis {...args} />
+);
+
+export const Default = Template.bind({});
+
+Default.args = {
+  dataArray: ndarray<number>(
+    dataset1.value.flat(Infinity) as number[],
+    dataset1.shape.dims.slice(0).reverse()
```

Comment:
Just transposing the dataset for a better visualization.

Reply:
Did you try [transpose](https://github.com/scijs/ndarray#arraytransposep0-p1-) instead ?

Anyway, I think the comment has its place in the code as well.